containers/haskell/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use the [Choice] comment to indicate option arguments that should appear in VS Code UX
+#
+# [Choice] Haskell version: 8,
```

Comment:
Yeah if you list the versions up above under `# [Choice]` VS Code UX will actually ask which version should be used.  For example, it looks like this could be as follows based on what I see in DockerHub.


```suggestion
# [Choice] Haskell version: 9, 8
```

"10" didn't seem published however.  Or you can list as many as you want really.